apps/docs/app/docs/components/tab/page.md

```diff
@@ -74,30 +74,41 @@ Pass `isUnstyled` prop in a particular subcomponent to remove style from that su
 
 {% example name="tab:unstyledsub" /%}
 
-### API
+## Props
 
 ---
 
-### Root
+### Tab
 
-| Properties | Description                                             | Type    | Default |
-| ---------- | ------------------------------------------------------- | ------- | ------- |
-| isUnstyled | Removes style from whole component and its subcomponent | boolean | false   |
+This component is built on top of [Radix Tabs](https://www.radix-ui.com/primitives/docs/components/tabs#root)
 
-### List
+| Properties  | Description                                             | Type                       | Default      |
+| ----------- | ------------------------------------------------------- | -------------------------- | ------------ |
+| size        | Size of the tab component.                              | `"sm"` or `"md"` or `"lg"` | `"md"`       |
+| isUnstyled  | Removes style from whole component and its subcomponent | boolean                    | `false`      |
+| orientation | Orientation of the component.                           | `horizontal` or `vertical` | `horizontal` |
```

Comment:
- Add double quotes on type and default value
- Change description to - "Tab orientation"